### PR TITLE
Access Control: Add fine-grained access control to ldap handlers

### PR DESCRIPTION
**Pull request by kalleep**
**What this PR does / why we need it**:
One handler for ldap was missing support for fgac.
Add tests for all ldap endpoint to validate permissions
Update docs with new permission

### CODE REVIEW

**Comment by osg-grafana** [comment]
In the title of this PR or issue, use an imperative verb to form a complete phrase.

File: docs/sources/enterprise/access-control/permissions.md [hunk 1]
```diff
@@ -61,6 +61,7 @@ Actions | Applicable scopes | Descriptions
 `ldap.user:read` | n/a | Get a user via LDAP.
 `ldap.user:sync` | n/a | Sync a user via LDAP.
 `ldap.status:read` | n/a | Verify the LDAP servers’ availability.
+`ldap.config:reload` | n/a | Reload LDAP config
```
Comment (osg-grafana) [inline docs/sources/enterprise/access-control/permissions.md:64]:
```suggestion
`ldap.config:reload` | n/a | Reload the LDAP configuration.
```
Write a complete sentence, and include a period at the end if need be.

File: docs/sources/enterprise/access-control/fine-grained-access-control-references.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ Fixed roles | Permissions | Descriptions
 `fixed:users:org:read` | `org.users:read` | Allows to get user organizations.
 `fixed:users:org:edit` | All permissions from `fixed:users:org:read` and <br>`org.users:add`<br>`org.users:remove`<br>`org.users.role:update` | Allows every read action for user organizations and in addition allows to administer user organizations.
 `fixed:ldap:admin:read` | `ldap.user:read`<br>`ldap.status:read` | Allows to read LDAP information and status.
-`fixed:ldap:admin:edit` | All permissions from `fixed:ldap:admin:read` and <br>`ldap.user:sync` | Allows every read action for LDAP and in addition allows to administer LDAP.
+`fixed:ldap:admin:edit` | All permissions from `fixed:ldap:admin:read` and <br>`ldap.user:sync`<br>`ldap.config:reload` | Allows every read action for LDAP and in addition allows to administer LDAP.
```
Comment (osg-grafana) [inline docs/sources/enterprise/access-control/fine-grained-access-control-references.md:24]:
```suggestion
`fixed:ldap:admin:edit` | All permissions from `fixed:ldap:admin:read` and <br>`ldap.user:sync`<br>`ldap.config:reload` | Read each action for LDAP, and administer LDAP.
```
What is the action here? Write a complete sentence, and include a period at the end if need be.

Comment (osg-grafana) [reply docs/sources/enterprise/access-control/fine-grained-access-control-references.md:24]: What does "Allows" mean here?

**Review by osg-grafana — CHANGES_REQUESTED** [review]

File: docs/sources/enterprise/access-control/fine-grained-access-control-references.md (hunk 2 above)
Comment (kalleep) [reply docs/sources/enterprise/access-control/fine-grained-access-control-references.md:24]: That was there before i updated it. I only added <br>`ldap.config:reload`

**Review by osg-grafana — APPROVED** [review]

**Comment by osg-grafana** [comment]
> In the title of this PR or issue, use an imperative verb to form a complete phrase.

Still interested in the answer to this question.

**Review by alexanderzobnin — APPROVED** [review]
Looks good!

**Pull request merged**